### PR TITLE
Feature/auth

### PR DESCRIPTION
**Pull request by ptesser**
### What type of request did you want to push:
- [X] Feature
- Bug
- Changelog
- Other

### Does this request introduce a breaking-change?
- yes
- [X] no
- maybe

### Other information:

This PR is blocked by `feature/change-app-name`. Need to merge that one before this.

### CODE REVIEW

File: app/app.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "name": "app",
-  "displayName": "app"
+  "name": "DrinkStop",
+  "displayName": "DrinkStop"
```
Comment (Vannnu) [inline app/app.json]: EOF

File: app/environments/environment.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export const environment = {
-  apiPath: 'http://localhost:8002/api/v1',
-}
+  apiAuthorizationKey: '',
+  apiPath: 'https://85c78a8e.ngrok.io',
+}
```
Comment (Vannnu) [inline app/environments/environment.ts]: EOF

Comment (Vannnu) [reply app/environments/environment.ts]: apiAuthorizationKey : empty?

Comment (ptesser) [reply app/environments/environment.ts]: Yes empty because I don't want to publish it on GitHub.

**Pull request merged**